packages/vscode/syntaxes/prisma-client-rawsql.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Prisma Client: raw SQL highlighting",
+  "scopeName": "client.rawsql.prisma",
+  "fileTypes": ["js", "jsx", "ts", "tsx", "vue"],
+  "injectionSelector": "L:source  -string -comment",
+  "patterns": [
+    {
+      "name": "rawSQLInTaggedTemplates",
+      "contentName": "meta.embedded.block.prisma-client-rawsql",
+      "begin": ".*(\\$queryRaw|\\$executeRaw|\\$queryRawUnsafe|\\$executeRawUnsafe)((`))",
+      "beginbegin": ".*\\$queryRaw((`))",
```

Comment:
What does this attribute do? (Docs link is fine)

Reply:
`beginbegin` should be removed, it was just here to debug 😄 
```suggestion
```

Reply:
And for the grammar it's https://macromates.com/manual/en/language_grammars